www/front_src/src/Resources/Filter/Criterias/Criteria.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { isNil } from 'ramda';
+
+import {
+  MultiAutocompleteField,
+  MultiConnectedAutocompleteField,
+  SelectEntry,
+} from '@centreon/ui';
+
+import { useStyles } from '..';
+import { useResourceContext } from '../../Context';
+import { labelOpen } from '../../translatedLabels';
+
+import { criteriaValueNameById, selectableCriterias } from './models';
+
+interface Props {
+  name: string;
+  value: Array<SelectEntry>;
+  parentWidth: number;
+}
+
+const Criteria = ({ name, value, parentWidth }: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const limitTags = parentWidth < 1000 ? 1 : 2;
+
+  const { setCriteria, setNewFilter } = useResourceContext();
+
+  const getTranslated = (values: Array<SelectEntry>): Array<SelectEntry> => {
+    return values.map((entry) => ({
+      id: entry.id,
+      name: t(entry.name),
+    }));
+  };
+
+  const changeCriteria = (updatedValue): void => {
+    setCriteria({ name, value: updatedValue });
+    setNewFilter();
+  };
+
+  const getUntranslated = (values): Array<SelectEntry> => {
```

Comment:
It’s used to convert back multi-select values to untranslated ones, because they are eventually saved in the database and in the local storage (as wall as the url query parameters). We dont’t want to have unmatched values in case the language is changed.